src/Sylius/Bundle/ShopBundle/Form/Extension/ChannelTypeExtension.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
```

Comment:
missing license block

Reply:
License block is missing. (I guess something happened at rebase)